pennylane/gradients/parameter_shift_hessian.py

```diff
@@ -704,8 +704,7 @@ def param_shift_hessian(tape, argnum=None, diagonal_shifts=None, off_diagonal_sh
 def _param_shift_hessian_tuple(
     tape, argnum=None, diagonal_shifts=None, off_diagonal_shifts=None, f0=None
 ):
-    r"""TODO: Update docstring examples
-
+    r"""
     Transform a QNode to compute the parameter-shift Hessian with respect to its trainable
```

Comment:
Do we still need the empty line? 

(i.e Should this rather be: `r""" Transform ...`)

Reply:
Either works.

Reply:
![image](https://user-images.githubusercontent.com/24476053/201193615-ff3602f9-351a-40ed-bd79-f436e745e4e7.png)